hardware/raspberrypi/bootmodes/msd.md

```diff
@@ -1,6 +1,24 @@
 # How to boot from a USB Mass Storage Device on a Raspberry Pi 3
 This tutorial explains how to boot your Raspberry Pi 3 from a USB mass storage device such as a flash drive or USB hard disk. Be warned that this feature is experimental and may not work with all USB mass storage devices.
 
+## Before Starting
+Check to make sure your SD card partitions are right for this example:
+```
+$ lsblk
+```
+NAME        MAJ:MIN RM  SIZE RO TYPE MOUNTPOINT
+mmcblk0     179:0    0  7.4G  0 disk 
+├─mmcblk0p1 179:1    0  2.3G  0 part 
+├─mmcblk0p2 179:2    0    1K  0 part 
+├─mmcblk0p5 179:5    0   32M  0 part /media/pi/SETTINGS
+├─mmcblk0p6 179:6    0   66M  0 part /boot
+└─mmcblk0p7 179:7    0    5G  0 part /
```

Comment:
I believe I now have article based around Raspbian rather than Noobs, but I have a section mentioning Noobs and how the sed commands would have to be changed.